backend/src/apiserver/server/util_test.go

```diff
@@ -108,90 +84,6 @@ func TestDecompressPipelineZip_MalformedZip2(t *testing.T) {
 	assert.Contains(t, err.Error(), "Not a valid zip file")
 }
 
-func TestDecompressPipelineZip_NonYamlZip(t *testing.T) {
```

Comment:
add a comment why it was removed to help with future KFP merges? I know there is a comment above, but that might not show in Git merge conflict resolution scope.